pop_tools/fill.py

```diff
@@ -183,3 +223,93 @@ def _iterative_fill_POP_core(nlat, nlon, var, fillmask, missing_value, tol, ltri
                             done = False
 
         var[1:nlat, :] = work[1:nlat, :]
+
+
+@jit(nopython=True)
```

Comment:
Use `parallel=True` so that Numba can compile a version that will run in parallel on multiple native threads

```suggestion
@jit(nopython=True, parallel=True)
```